openc3-init/plugins/packages/openc3-tool-limitsmonitor/src/tools/LimitsMonitor/LimitsControl.vue

```diff
@@ -204,6 +207,27 @@ export default {
       }
     }
     this.updateOutOfLimits()
+
+    this.cable
+      .createSubscription('LimitsEventsChannel', localStorage.scope, {
+        received: (data) => {
+          const parsed = JSON.parse(data)
+          this.handleMessages(parsed)
+        },
+      })
+      .then((limitsSubscription) => {
+        this.limitsSubscription = limitsSubscription
+      })
```

Comment:
Moving the subscription here is duplication of the Events subscription but greatly simplifies the design since the Events wants history